groups/groups.yaml

```diff
@@ -171,6 +171,18 @@ groups:
       - spiffxp@google.com
       - thockin@google.com
 
+  - email-id: k8s-infra-artifact-security@kubernetes.io
+    name: k8s-infra-artifact-admins
```

Comment:
I guess it doesn't matter that the `name` field is the same as line 161?

Reply:
```suggestion
    name: k8s-infra-artifact-security
```

Reply:
ACK